test/indexer/data/fa5be5eb-2d64-49f5-8ed8-bd627ac9bc7a.metadata.json

```diff
@@ -393,7 +407,14 @@
         "file_core": {
             "file_name": "pbmc4k_1000_S1_L001_I1_001.fastq.gz",
             "file_format": "fastq.gz",
-            "checksum": "3d252f83492bf030ddfd05a5a1411cdc"
+            "checksum": "3d252f83492bf030ddfd05a5a1411cdc",
+            "content_description": [
```

Comment:
The problem is that the [declared schema](http://schema.staging.data.humancellatlas.org/type/file/7.0.2/sequence_file) does not allow this. Can we cover this differently? We should also cover the grouping aspect. There are a few canned bundles that already have `content_description` so maybe we need to bite the bullet and add a test specifically for `fileTypeSummaries`.